updatecli/updatecli.d/installation.yaml

```
@@ -0,0 +1,51 @@
+name: Bump build-tekton utils reference
+
+scms:
+  gitjob:
+    kind: github
+    spec:
+      user: gitjob-bot
+      email: gitjob@olblak.com
+      owner: rancher
+      repository: gitjob
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "UPDATECLI_GITHUB_ACTOR" }}'
+      branch: master
+
+sources:
+  build-tekton:
+    name: Get latest build-tekton version
+    kind: githubrelease
+    spec:
+      owner: rancher
+      repository: build-tekton
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "UPDATECLI_GITHUB_ACTOR" }}'
+      versionfilter:
+        kind: semver
+    # The assets name do not contains the 'v' prefix before the version
+    transformers:
+      - trimprefix: v
+
+targets:
+  values-yaml:
+    name: 'Update build-tekton reference'
+    kind: file
+    spec:
+      file: chart/values.yaml
+      matchpattern: 'tag: v(\d)(.*)'
+      replacepattern: 'tag: v{{ source "build-tekton" }}'
```

```suggestion
      replacepattern: 'tag: {{ source "build-tekton" }}'
```

If you like my comment below then this transformers should be remove so to keep the v prefix if it's defined